### PR TITLE
Release guid from removed pods

### PR DESCRIPTION
**Pull request by Mmduh-483**

### CODE REVIEW

File: pkg/daemon/daemon.go [hunk 1]
```diff
@@ -234,6 +234,8 @@ func (d *daemon) AddPeriodicUpdate() {
 					glog.Errorf("AddPeriodicUpdate(): failed to update pod annotations with err: %v", err)
 					continue
 				}
+
+				_ = d.guidPool.ReleaseGUID(guidList[index].String())
```
Comment (moshe010) [inline pkg/daemon/daemon.go]: if it failed just log a warning the we will know

Comment (Mmduh-483) [reply pkg/daemon/daemon.go]: added

File: pkg/daemon/daemon.go [hunk 2]
```diff
@@ -234,6 +234,11 @@ func (d *daemon) AddPeriodicUpdate() {
 					glog.Errorf("AddPeriodicUpdate(): failed to update pod annotations with err: %v", err)
 					continue
 				}
+
+				if err = d.guidPool.ReleaseGUID(guidList[index].String()); err != nil {
+					glog.Warningf("AddPeriodicUpdate(): failed to release guid from removed pod with error: %v",
```
Comment (moshe010) [inline pkg/daemon/daemon.go]: please print guid and pod so it will be easy to debug

Comment (Mmduh-483) [reply pkg/daemon/daemon.go]: added

**Pull request merged**